src/supervised/distance.jl

```diff
@@ -145,29 +153,43 @@ isstronglyconvex(::PeriodicLoss) = false
 # ===========================================================
 # L(y, t) = max(0, |y - t| - ɛ)
 
-immutable L1EpsilonInsLoss <: DistanceLoss
-    eps::Float64
+immutable L1EpsilonInsLoss{T<:AbstractFloat} <: DistanceLoss
+    ε::T
```

Comment:
Do pings work in comments? I seem to remember someone having problems with this.

I believe users should be comfortable with unicode, this is what Distributions.jl does. When I was first starting out using Julia it tripped me up for a little bit, I maybe got a little frustrated, but then I realized it was 100% the way to go and I was a fool for thinking otherwise.

My real reasoning for changing this was `eps` can be confused with `Base.eps()`. So if there is an alternative I can happily put that in.